cmd/tempo-serverless/cloud-run/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/grafana/tempo/cmd/tempo-serverless/cloud-run
 
-go 1.21
+go 1.22
 
-toolchain go1.21.0
```

Comment:
is the `.0` important? would it be better to specify the minor version only and let the patch float?